python/mxnet/optimizer.py

```diff
@@ -157,7 +157,7 @@ def set_wd_mult(self, args_wd_mult):
         """
         self.wd_mult = {}
         for n in self.idx2name.values():
-            if not n.endswith('_weight'):
+            if not (n.endswith('_weight') or n.endswith('_gamma')):
```

Comment:
@piiswrong @mavenlin @vchuravy 
decaying is applied on weight and bias( gamma and beta is regarded as weight and bias ) in torch
reference
https://github.com/torch/optim/blob/master/sgd.lua
https://github.com/torch/nn/blob/master/BatchNormalization.lua
https://github.com/facebook/fb.resnet.torch/